etc/profile-m-z/steam.profile

```diff
@@ -9,6 +9,7 @@ include globals.local
 noblacklist ${HOME}/.killingfloor
 noblacklist ${HOME}/.local/share/3909/PapersPlease
 noblacklist ${HOME}/.local/share/aspyr-media
+noblacklist ${HOME}/.local/share/bohemiainteractive
```

Comment:
can you add the corresponding
`blacklist ${HOME}/.local/share/bohemiainteractive`
to etc/disable-programs.inc

Reply:
Added.